packages/salesforcedx-vscode-core/test/vscode-integration/util/statuses/taskView.test.ts

```diff
@@ -29,27 +29,21 @@ describe('Task View', () => {
     it('Should add a command to its internal queue', () => {
       taskViewService.addCommandExecution(
         new CliCommandExecutor(
-          new SfdxCommandBuilder()
-            .withArg('force')
-            .withArg('--help')
-            .build(),
+          new SfCommandBuilder().withArg('force').withArg('--help').build(),
```

Comment:
Same question as above - why are we still running `sf force --help`?  What is the purpose of running it?